src/styles/components/search-field.css

```diff
@@ -0,0 +1,3 @@
+.pos-search-field:focus-within {
+  @apply border-interactive-graphic;
```

Comment:
you can use the `focus-within:border-interactive-graphic` class in your component